src/R/Components/Impl/InteractiveWorkflow/Implementation/RSessionCallback.cs

```diff
@@ -61,25 +61,41 @@ internal sealed class RSessionCallback : IRSessionCallback {
         /// Displays R plot in the host app-provided window
         /// </summary>
         public async Task Plot(PlotMessage plot, CancellationToken ct) {
```

Comment:
Note that this is a callback corresponding to a host message. Our naming convention for those so far has been to use the same name as the message (mostly), without `Async` - this is stemming from the fact that the first callbacks were all corresponding to functions that R calls, like `ReadConsole`, and the desire to establish a clear mapping between those functions, and corresponding C# callbacks.